entity-framework/core/miscellaneous/events/dotnet-events.md

```diff
@@ -0,0 +1,133 @@
+---
+title: .NET events - EF Core
+description: .NET events defined by EF Core  
+author: ajcvickers
+ms.date: 10/15/2020
+uid: core/miscellaneous/events/dotnet-events
+---
+
+# .NET events in EF Core
+
+> [!TIP]  
+> You can [download the events sample](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/Miscellaneous/Events) from GitHub.
+
+Entity Framework Core (EF Core) exposes [.NET events](/dotnet/standard/events/) to act as callbacks when certain things happen in the EF Core code. Events are simpler than [interceptors](xref:core/miscellaneous/events/interception) and allow more flexible registration. However, they are sync only and so cannot perform non-blocking async I/O.
+
+Events are registered per `DbContext` instance. Use a [diagnostic listener](xref:core/miscellaneous/events/diagnostics) to get the same information but for all DbContext instances in the process.
```

Comment:
Maybe make even clearer the fact that events can be used per-instance, unlike interceptors.